distribution/docs/src/main/resources/draft-documentation.adoc

```diff
@@ -73,54 +73,102 @@ include::{adoc-include}/_introduction/managing-intro-contents.adoc[]
 
 include::{adoc-include}/_installing/installing-intro-contents.adoc[]
 
-//configuring
+include::{adoc-include}/_installing/controlling-file-system-access-contents.adoc[]
+
+include::{adoc-include}/_installing/initial-startup-contents.adoc[]
+
+include::{adoc-include}/_installing/installing-from-admin-console-contents.adoc[]
+
+include::{adoc-include}/_installing/configuring-guest-claims-attributes-contents.adoc[]
+
+include::{adoc-include}/_installing/ddf-setup-types-contents.adoc[]
+
+include::{adoc-include}/_installing/system-configuration-settings-contents.adoc[]
+
+include::{adoc-include}/_installing/finish-page-contents.adoc[]
+
 == Configuring
 
 include::{adoc-include}/_configuring/configuring-intro-contents.adoc[]
 
+include::{adoc-include}/_securing/securing-intro-contents.adoc[]
+
+include::{adoc-include}/_securing/hardening-intro-contents.adoc[]
+
+<<<
+include::{adoc-include}/_securing/hardening-checklist-contents.adoc[]
+<<<
+
+include::{adoc-include}/_securing/managing-certificates-contents.adoc[]
+
 === Configuring from the ${admin-console}
 
 include::{adoc-include}/_configuring/admin-console-intro-contents.adoc[]
 
-include::{adoc-include}/_configuring/configuring-from-the-admin-console-contents.adoc[]
+include::{adoc-include}/_configuring/admin-console-tutorial-contents.adoc[]
+////
+TODO: document configuration Wizard
```

Comment:
Since it's hidden to the user, I'd vote to leave it in so when 2.11 comes around, it has a place waiting for it.